core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -171,6 +171,20 @@ class CityInfo {
         return cityResources
     }
 
+    fun getCityUntappedResources(): ResourceSupplyList {
+        val cityResources = ResourceSupplyList()
+
+        // count missed chances: resource is known, tile owned, but not improved or improvement not matching / not active
+        for (tileInfo in getTiles().filter { it.resource != null }) {
+            val resource = tileInfo.getTileResource()
+            if (resource.resourceType==ResourceType.Bonus) continue
+            if (resource.revealedBy!=null && !civInfo.tech.isResearched(resource.revealedBy!!)) continue
+            if (getTileResourceAmount(tileInfo)==0)
+                cityResources.add(resource, -1, "Untapped")
```

Comment:
It still confuses me, why it is counted as a negative.
"I have -5 untapped resources." - that sounds strange.
I would suggest to display them as a positive number in braces to indicate that it is not counted to the total.

For example:
![Screenshot 2020-03-20 08 11 36](https://user-images.githubusercontent.com/27405436/77140492-73d73d80-6a82-11ea-81c5-ca0ed710f78b.png)

Reply:
You're right. The technical negative is left over from abusing it as a flag, the display was conscious but I did hesitate. (n) sounds good. I fear commiting that may pose surprises as I merged 3.6.9 into my masters, and that branch is based on 3.6.8. I'll try in a few hours.